Tests/SentryTests/SentryFormatterTests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+
+final class SentryFormatterTests: XCTestCase {
+    func testFormatHexAddress() {
+        for (input, expected) in [
+            (0x000000008e902bf0, "0x000000008e902bf0"),
+            (0x000000008fd09c40, "0x000000008fd09c40"),
+            (0x00000000945b1c00, "0x00000000945b1c00")
+        ] {
+            XCTAssertEqual(sentry_formatHexAddress(input as NSNumber), expected)
+        }
+    }
+
+    func testStringForUInt64() {
+        for (input, expected) in [
+            (0, "0"),
+            (1, "1"),
+            (123_456, "123456"),
+            (UInt64.max, "18446744073709551615")
+        ] {
+            XCTAssertEqual(sentry_stringForUInt64(UInt64(input)), expected)
+        }
+    }
+
+    func testFormatHexAddressPerformance() {
```

Comment:
`m`: `Measure` doesn't work in CI, as the baselines only work on the same computer config as you set them. If two engineers have the same MacBook with the same CPU and RAM, `measure` will fail if some code gets too slow. You didn't commit any baseline files, so it wouldn't work anyway. This test now only slows down CI, so we could also remove it.

Reply:
Ah yeah, I only had this in there to compare some alternatives to snprintf. I;ll remove this.